configuration/initialization.sh

```diff
@@ -9,15 +87,15 @@ sudo apt update -y && sudo apt upgrade -y
 sudo apt-get install chromium-browser -y
 
 # Make passwords for mysql
-mkdir -p secrets && cd secrets
+mkdir -p secrets && cd secrets || return
```

Comment:
Why is this necessary. Is this something from shell check. Can you add justification for adding return statement here, maybe even link rule.

Reply:
Yes, shell has thrown a warning here "^-- [SC2164](https://www.shellcheck.net/wiki/SC2164) (warning): Use 'cd ... || exit' or 'cd ... || return' in case cd fails."

Added a comment next to return saying "# return in case cd fails"